src/v2/view-builder/views/SuccessView.js

```diff
@@ -8,10 +8,13 @@ const Body = BaseForm.extend({
   noButtonBar: true,
   initialize () {
     BaseForm.prototype.initialize.apply(this, arguments);
+    const currentViewState = this.options.appState.getCurrentViewState();
+    if (currentViewState.name === 'device-apple-sso-extension') {
+      document.cookie = `stateHandle=${this.options.appState.get('currentState').stateHandle};path=/`;
```

Comment:
- would be better to keep `SSOExtensionView.js` given this specific logic.
- please add comment regarding why the sso extension flow is so special.

Reply:
@haishengwu-okta , `SSOExtensionView.js` is back, but not extended from SuccessView as SuccessView.Body is not accessible.